### ISSUE TITLE
Array Index support ie: $users[0]->name

### ISSUE THREAD

**Issue by brad-jones** [php-integrator/atom-autocompletion#81]
Continuing on from #33 I am happy that the following auto-completes:

```php
foreach($classes as $myClassInstance) {
    $myClassInstance->hel...
}
```

But I would expect the following to also autocomplete:

```php
$classes[0]->hel...
$classes[1]->hel...
$classes[5]->hel...
```

And it does not appear to.

**Comment by Gert-dev** [comment]
You are right, I've actually already implemented this not so long ago and it will work in the soon-to-be-released 2.0.0.

**Comment by Gert-dev** [comment]
I've just released 2.0, this should now be working. If it isn't for some reason, feel free to reply.
